GameStudies.js

```diff
@@ -0,0 +1,108 @@
+/*
+Game Studies
+Copyright (C) 2015 John Weis
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+
+function detectWeb(doc, url) { 
+	return FW.detectWeb(doc, url); }
+function doWeb(doc, url) { return FW.doWeb(doc, url); }
+
+
+
+FW.Scraper({
+  itemType			:'journalArticle',
+  detect			:FW.Url().match(/gamestudies\.org\/\d+\/articles\/\w+/),
+  title				:FW.Xpath('//div[@id="article"]/h2').text().trim(),
+  creators			:FW.Xpath('//div[@id="article"]/small').text().remove(/by/).cleanAuthor("author"),
+  publicationTitle	:"Game Studies",
+  abstractNote		:FW.Xpath('//div[@id="article"]/p[1]').text().trim(),
+  date				:FW.Xpath('//span[@class="date"]').text(),
+  volume			:FW.Xpath('//span[@class="volume"]').text().match(/volume\s(\d+)/, 1),
+  issue 			:FW.Xpath('//span[@class="issueno"]').text().match(/issue\s(\d+)/, 1),
+  tags				:FW.Xpath('/html/body/div/div[3]/div/p[2]').text().split(/,/).trim(),
```

Comment:
remove the "Keywords:" at the beginning, something like ```.remove(/^\s*Keywords:\s*/)```.
Also, I'd try to make this a little more stable by doing something like ```'//div[@id="article"]/p[2]'``` instead of the full xpath.